File: codebattle/python/get_alphabet.py
#Задача: get_alphabet

#Return an alphabet of a given string; the given string is represented as an array of characters. An alphabet of a strin is the set of all distinct characters used in that string.

#Example: "adfs" == solution("asfsfdss")

#https://github.com/Hexlet/battle_asserts/blob/master/src/battle_asserts/issues/get_alphabet.clj

# BEGIN implement function `solution` here 
def solution(s):
  return ''.join(sorted(''.join(set(s))))
# END

File: codebattle/php/get_alphabet.php
<?php

//Задача: get_alphabet

//Return an alphabet of a given string; the given string is represented as an array of characters. An alphabet of a strin is the set of all distinct characters used in that string.

//Example: "adfs" == solution("asfsfdss")

//https://github.com/h/b/blob/master/src/b/issues/get_alphabet.clj

 
function solution($a) {
    $alpha = [];
    array_map(function($x) use(&$alpha) {
    	if (!in_array($x, $alpha))
            $alpha[] = $x;
    }, str_split($a));
    sort($alpha);
    return implode('', $alpha);
}
